ansible/group_vars/all.yml

```diff
@@ -65,7 +69,7 @@ virtual_machines:
         users:
           - name: core
             sshAuthorizedKeys:
-              - "{{ lookup('file', gcp_ssh_key_file + '.pub') }}"
+              - "{{ lookup('file', gcp_ssh_key_file + '.pub', errors='ignore') }}"
```

Comment:
This is needed for a somewhat convoluted reason. When building the collector images, `virtual_machines` was not evaluated because it wasn't used and so this line was ignored, now `virtual_machines` is needed for both `arch_info` and `ansible_ssh_private_key_file`, so this line is evaluated and fails. That file is only really needed for RHCOS testing though, so ignoring the missing file should be fine.